File: scripts/extract_pdf_text.js
const fs = require('fs');
const pdf = require('pdf-parse');
const inPath = 'c:\\Users\\krish\\OneDrive\\Desktop\\Karagir\\temp\\karagir editor.pdf';
const outPath = 'c:\\Users\\krish\\OneDrive\\Desktop\\Karagir\\temp\\karagir_editor.txt';
let dataBuffer = fs.readFileSync(inPath);
pdf(dataBuffer).then(function(data) {
  fs.writeFileSync(outPath, data.text, 'utf8');
  console.log('WROTE', outPath);
}).catch(err => { console.error(err); process.exit(1); });

File: scripts/extract_pdf_text.py
from pdfminer.high_level import extract_text
pdf_path = r'c:\Users\krish\OneDrive\Desktop\Karagir\temp\karagir editor.pdf'
out_path = r'c:\Users\krish\OneDrive\Desktop\Karagir\temp\karagir_editor.txt'
text = extract_text(pdf_path)
with open(out_path, 'w', encoding='utf-8') as f:
    f.write(text)
print('WROTE', out_path)
